### PR TITLE
DO alignment (partial) - 230314

### PR DESCRIPTION
**Pull request by sabrinatoro**

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -463641,6 +463697,48 @@ xref: DOID:0080619 {source="MONDO:equivalentTo"}
...
+[Term]
+id: MONDO:0850154
+name: tongue carcinoma
+xref: DOID:0080641 {source="MONDO:equivalentTo"}
+is_a: MONDO:0004631 ! tongue cancer
+is_a: MONDO:0004993 ! carcinoma
+
+[Term]
+id: MONDO:0850156
+name: b-lymphoblastic leukemia/lymphoma mll rearranged
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:463709]: B-cells should be capitalized, can you fix this for all of them?

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -463641,6 +463697,48 @@ xref: DOID:0080619 {source="MONDO:equivalentTo"}
...
+def: "A B-lymphoblastic leukemia/lymphoma that is characterized by the presence of lymphoblasts that carry a translocation between the KMT2A gene at 11q23.3 and another gene partner resulting in the production of a KMT2A related fusion protein." [DOID:0080644]
+xref: DOID:0080644 {source="MONDO:equivalentTo"}
+is_a: MONDO:0004947 {source="DOID:0080644"} ! B-cell acute lymphoblastic leukemia
+
+[Term]
+id: MONDO:0850157
+name: b-lymphoblastic leukemia/lymphoma with etv6-runx1
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:463716]: the gene names should be capitalized, ETV6-RUNX1

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -463641,6 +463697,48 @@ xref: DOID:0080619 {source="MONDO:equivalentTo"}
...
+def: "A B-lymphoblastic leukemia/lymphoma that is characterized by the presence of lymphoblasts that carry a translocation between the TEL gene on chromosome 12 and the AML1 gene on chromosome 21, (p13.2;q22.1). It results in the production of the TEL-AML1 (ETV6-RUNX1) fusion protein." [DOID:0080645]
+xref: DOID:0080645 {source="MONDO:equivalentTo"}
+is_a: MONDO:0004947 {source="DOID:0080645"} ! B-cell acute lymphoblastic leukemia
+
+[Term]
+id: MONDO:0850160
+name: b-lymphoblastic leukemia/lymphoma with il3-igh
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:463723]: - B should be capital
- IL3 should be capital
- is 'igh' a typo? should be high?

File: src/ontology/mondo-edit.obo [hunk 4]
```diff
@@ -463641,6 +463697,48 @@ xref: DOID:0080619 {source="MONDO:equivalentTo"}
...
+def: "A B-lymphoblastic leukemia/lymphoma that is characterized by the presence of lymphoblasts that carry a translocation between the IL3 gene on chromosome 5 and the IGH locus on chromosome 14, (q31.1;q32.3)." [DOID:0080648]
+xref: DOID:0080648 {source="MONDO:equivalentTo"}
+is_a: MONDO:0004947 {source="DOID:0080648"} ! B-cell acute lymphoblastic leukemia
+
+[Term]
+id: MONDO:0850161
+name: b-lymphoblastic leukemia/lymphoma, bcr-abl1–like
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:463730]: - B should be capital
- BCR-ABL1 should be capital

File: src/ontology/mondo-edit.obo [hunk 5]
```diff
@@ -463675,6 +463789,13 @@ def: "A cognitive disorder that is characterized by the presence of significant
 xref: DOID:0080831 {source="MONDO:equivalentTo"}
 is_a: MONDO:0002039 {source="DOID:0080831"} ! cognitive disorder
 
+[Term]
+id: MONDO:0850295
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:463793]: can you pls add the logical def according to the acquired pattern?
thanks!

File: src/ontology/mondo-edit.obo [hunk 6]
```diff
@@ -463699,6 +463820,35 @@ xref: DOID:0080901 {source="MONDO:equivalentTo"}
...
+[Term]
+id: MONDO:0850371
+name: nonobstructive coronary artery disease
+def: "A coronary artery disease that is characterized by atherosclerotic plaque that would not be expected to obstruct blood flow or result in anginal symptoms and stenosis of coronary artery less than 50 percent." [DOID:0080938]
+xref: DOID:0080938 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005010 {source="DOID:0080938"} ! coronary artery disorder
+
+[Term]
+id: MONDO:0850388
+name: childhood supratentorial embryonal tumor with multilayered rosettes, c19mc-altered
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:463839]: - should be C19MC

File: src/ontology/mondo-edit.obo [hunk 7]
```diff
@@ -463699,6 +463820,35 @@ xref: DOID:0080901 {source="MONDO:equivalentTo"}
...
+def: "A childhood embryonal tumor with multilayered rosettes, C19MC-altered that arises from the supratentorial brain and occurs in children." [DOID:0080956]
+xref: DOID:0080956 {source="MONDO:equivalentTo"}
+is_a: MONDO:0004378 {source="DOID:0080956"} ! pediatric cerebral ependymoblastoma
+relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/general/qc-single-child.sparql
+
+[Term]
+id: MONDO:0850426
+name: high-grade b-cell lymphoma double-hit/triple-hit
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:463847]: B should be capital

File: src/ontology/mondo-edit.obo [hunk 8]
```diff
@@ -463727,6 +463877,13 @@ xref: DOID:0081121 {source="MONDO:equivalentTo"}
 is_a: MONDO:0000426 {source="DOID:0081121"} ! autosomal dominant disease
 is_a: MONDO:0000507 {source="DOID:0081121"} ! inclusion body myopathy with Paget disease of bone and frontotemporal dementia
 
+[Term]
+id: MONDO:0850519
+name: tetrahydrobiopterin (bh4)-deficient hyperphenylalaninemia
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:463882]: BH4 should be capital

**Review by nicolevasilevsky — CHANGES_REQUESTED** [review]

File: src/ontology/mondo-edit.obo (hunk 5 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:463793]: added the subclassOf axiom and not the logical def because there is no term for "laryngomalacia"

**Review by nicolevasilevsky — APPROVED** [review]

**Pull request merged**